Session3.Rmd

```diff
@@ -576,7 +586,7 @@ RGB_band1_TUD_df  <- as.data.frame(RGB_band1_TUD, xy = TRUE)
 ```{r}
 ggplot() +
   geom_raster(data = RGB_band1_TUD_df,
-              aes(x = x, y = y, alpha = tudlib.rgb)) + 
+              aes(x = x, y = y, alpha = tudlib.rgb_1)) + 
```

Comment:
I think i had this issue the last time around :  my `RGB_band1_TUD_df` dataset has three variables: `x`, `y`, and `layer` ( not  `tudlib.rgb_1`)